src/index.spec.ts

```diff
@@ -802,6 +804,31 @@ describe('ts-node', function () {
           return done()
         })
       })
+
```

Comment:
Nitpick: this suite name might imply that the ESM loader only supports transpile-only via this entrypoint, though it can also load this flag from environment variables or tsconfig.json.

Maybe name the 2x test cases to match some of our other specs' names, and remove the `describe()` wrapper?
```
it('should support transpile only mode via dedicated loader entrypoint', ...
it('should throw errors', ...
```

Reply:
Is this acceptable?
https://github.com/TypeStrong/ts-node/blob/525b7308fc45355fa46f6306c36895e6de19c2fd/src/index.spec.ts#L808-L830

Reply:
Yeah, looks good